tests/workflowRun/fixtures/metrics.ts

```diff
@@ -202,7 +202,7 @@ export const exampleJobMetrics: v1.Series[] = [
   {
     host: 'github.com',
     metric: 'github.actions.job.queued_duration_second',
-    points: [[1579542279, 0]],
+    points: [[1579542279, -179428]],
```

Comment:
Is this value supposed to be negative?

Reply:
It's just because their data is all over the place. 

Here is the Workflow run:
https://github.com/iFixit/datadog-actions-metrics/blob/main/tests/workflowRun/fixtures/workflowRunEvent.ts#L60:
![image](https://user-images.githubusercontent.com/29667789/186261647-9b1be51f-605c-43e2-975a-9ff515c514cd.png)

but the job started two days before:
https://github.com/iFixit/datadog-actions-metrics/blob/main/tests/workflowRun/fixtures/completedCheckSuite.ts#L36
![image](https://user-images.githubusercontent.com/29667789/186261919-91775407-c51b-43e1-a06e-a2882fe5d65e.png)

In reality, that would never happen. I could update all of the tests but that seems like a lot of work.

---

You can see the workflow queued at value is negative b/c weird data:
https://github.com/iFixit/datadog-actions-metrics/blob/main/tests/workflowRun/fixtures/metrics.ts#L58-L76

Reply:
Ahh that sucks, but gotcha 👍